### ISSUE TITLE
VirFinder with specialized model for euks

### ISSUE THREAD

**Issue by hoelzer** [hoelzer/virify#21]
see
https://github.com/guille0387/Cabana-secondment/blob/master/VirFinder_analysis_Euk.R

model:
https://github.com/jessieren/VirFinder/raw/master/EPV/VF.modEPV_k8.rda

**Comment by hoelzer** [comment]
inside the docker:
```
cp /VirFinder/EPV/VF.modEPV_k8.rda /usr/local/lib/R/site-library/VirFinder/data/
```

**Comment by hoelzer** [comment]
not really needed, the rda file can be also passed to the script directly

**Comment by hoelzer** [comment]
```
      wget https://github.com/jessieren/VirFinder/raw/master/EPV/VF.modEPV_k8.rda
      run_virfinder_modEPV.Rscript VF.modEPV_k8.rda ${fasta} .
```

**Comment by hoelzer** [comment]
Implemented in PR #22
